### ISSUE TITLE
Are the quoted numbers accurate in the cifar-10 script?

### ISSUE THREAD

**Issue by LetterRip** [fchollet/keras#2937]
The numbers seem 'optomistic' for the cifar-10,  after 70 epochs I'm nowhere close to the numbers stated in the script claims to get at 40 epoch and reading bug reports - other people tend to get a NN with similar accuracy (80-81%).

Here is the output for all 71 epochs

50000/50000 [==============================] - 678s - loss: 1.7646 - acc: 0.3460 - val_loss: 1.3379 - val_acc: 0.5161
Epoch 2/200
50000/50000 [==============================] - 701s - loss: 1.3731 - acc: 0.5027 - val_loss: 1.0936 - val_acc: 0.6131
Epoch 3/200
50000/50000 [==============================] - 706s - loss: 1.2137 - acc: 0.5645 - val_loss: 1.0011 - val_acc: 0.6478
Epoch 4/200
50000/50000 [==============================] - 709s - loss: 1.1154 - acc: 0.6033 - val_loss: 0.8915 - val_acc: 0.6866
Epoch 5/200
50000/50000 [==============================] - 708s - loss: 1.0465 - acc: 0.6294 - val_loss: 0.8605 - val_acc: 0.7020
Epoch 6/200
50000/50000 [==============================] - 707s - loss: 1.0045 - acc: 0.6450 - val_loss: 0.8188 - val_acc: 0.7131
Epoch 7/200
50000/50000 [==============================] - 706s - loss: 0.9582 - acc: 0.6624 - val_loss: 0.7967 - val_acc: 0.7213
Epoch 8/200
50000/50000 [==============================] - 707s - loss: 0.9401 - acc: 0.6698 - val_loss: 0.7908 - val_acc: 0.7217
Epoch 9/200
50000/50000 [==============================] - 706s - loss: 0.9125 - acc: 0.6786 - val_loss: 0.7928 - val_acc: 0.7213
Epoch 10/200
50000/50000 [==============================] - 712s - loss: 0.9029 - acc: 0.6867 - val_loss: 0.7373 - val_acc: 0.7464
Epoch 11/200
50000/50000 [==============================] - 703s - loss: 0.8805 - acc: 0.6929 - val_loss: 0.7339 - val_acc: 0.7488
Epoch 12/200
50000/50000 [==============================] - 705s - loss: 0.8716 - acc: 0.6975 - val_loss: 0.7351 - val_acc: 0.7474
Epoch 13/200
50000/50000 [==============================] - 704s - loss: 0.8701 - acc: 0.6980 - val_loss: 0.7118 - val_acc: 0.7592
Epoch 14/200
50000/50000 [==============================] - 705s - loss: 0.8568 - acc: 0.7026 - val_loss: 0.7170 - val_acc: 0.7509
Epoch 15/200
50000/50000 [==============================] - 706s - loss: 0.8455 - acc: 0.7070 - val_loss: 0.7144 - val_acc: 0.7547
Epoch 16/200
50000/50000 [==============================] - 706s - loss: 0.8402 - acc: 0.7092 - val_loss: 0.7077 - val_acc: 0.7587
Epoch 17/200
50000/50000 [==============================] - 704s - loss: 0.8281 - acc: 0.7142 - val_loss: 0.7092 - val_acc: 0.7572
Epoch 18/200
50000/50000 [==============================] - 709s - loss: 0.8237 - acc: 0.7141 - val_loss: 0.7385 - val_acc: 0.7435
Epoch 19/200
50000/50000 [==============================] - 708s - loss: 0.8221 - acc: 0.7187 - val_loss: 0.6947 - val_acc: 0.7653
Epoch 20/200
50000/50000 [==============================] - 708s - loss: 0.8065 - acc: 0.7200 - val_loss: 0.6668 - val_acc: 0.7746
Epoch 21/200
50000/50000 [==============================] - 705s - loss: 0.8116 - acc: 0.7191 - val_loss: 0.6542 - val_acc: 0.7782
Epoch 22/200
50000/50000 [==============================] - 705s - loss: 0.8026 - acc: 0.7230 - val_loss: 0.6961 - val_acc: 0.7628
Epoch 23/200
50000/50000 [==============================] - 705s - loss: 0.8079 - acc: 0.7233 - val_loss: 0.6516 - val_acc: 0.7763
Epoch 24/200
50000/50000 [==============================] - 705s - loss: 0.7935 - acc: 0.7284 - val_loss: 0.6369 - val_acc: 0.7874
Epoch 25/200
50000/50000 [==============================] - 704s - loss: 0.7916 - acc: 0.7296 - val_loss: 0.6373 - val_acc: 0.7835
Epoch 26/200
50000/50000 [==============================] - 702s - loss: 0.7996 - acc: 0.7254 - val_loss: 0.6692 - val_acc: 0.7773
Epoch 27/200
50000/50000 [==============================] - 704s - loss: 0.7945 - acc: 0.7272 - val_loss: 0.6274 - val_acc: 0.7884
Epoch 28/200
50000/50000 [==============================] - 705s - loss: 0.7864 - acc: 0.7287 - val_loss: 0.6561 - val_acc: 0.7817
Epoch 29/200
50000/50000 [==============================] - 703s - loss: 0.7896 - acc: 0.7303 - val_loss: 0.6403 - val_acc: 0.7841
Epoch 30/200
50000/50000 [==============================] - 700s - loss: 0.7778 - acc: 0.7341 - val_loss: 0.6983 - val_acc: 0.7614
Epoch 31/200
50000/50000 [==============================] - 701s - loss: 0.7801 - acc: 0.7352 - val_loss: 0.6287 - val_acc: 0.7870
Epoch 32/200
50000/50000 [==============================] - 701s - loss: 0.7806 - acc: 0.7341 - val_loss: 0.6288 - val_acc: 0.7924
Epoch 33/200
50000/50000 [==============================] - 694s - loss: 0.7721 - acc: 0.7347 - val_loss: 0.6406 - val_acc: 0.7851
Epoch 34/200
50000/50000 [==============================] - 690s - loss: 0.7772 - acc: 0.7353 - val_loss: 0.6186 - val_acc: 0.7913
Epoch 35/200
50000/50000 [==============================] - 689s - loss: 0.7712 - acc: 0.7363 - val_loss: 0.6650 - val_acc: 0.7773
Epoch 36/200
50000/50000 [==============================] - 799s - loss: 0.7784 - acc: 0.7361 - val_loss: 0.6450 - val_acc: 0.7800
Epoch 37/200
50000/50000 [==============================] - 760s - loss: 0.7757 - acc: 0.7350 - val_loss: 0.6757 - val_acc: 0.7732
Epoch 38/200
50000/50000 [==============================] - 689s - loss: 0.7772 - acc: 0.7340 - val_loss: 0.6248 - val_acc: 0.7885
Epoch 39/200
50000/50000 [==============================] - 691s - loss: 0.7653 - acc: 0.7400 - val_loss: 0.6317 - val_acc: 0.7881
Epoch 40/200
50000/50000 [==============================] - 688s - loss: 0.7655 - acc: 0.7398 - val_loss: 0.6243 - val_acc: 0.7906
Epoch 41/200
50000/50000 [==============================] - 686s - loss: 0.7655 - acc: 0.7403 - val_loss: 0.6436 - val_acc: 0.7807
Epoch 42/200
50000/50000 [==============================] - 685s - loss: 0.7645 - acc: 0.7391 - val_loss: 0.6266 - val_acc: 0.7916
Epoch 43/200
50000/50000 [==============================] - 681s - loss: 0.7711 - acc: 0.7381 - val_loss: 0.6114 - val_acc: 0.7953
Epoch 44/200
50000/50000 [==============================] - 679s - loss: 0.7626 - acc: 0.7413 - val_loss: 0.6245 - val_acc: 0.7912
Epoch 45/200
50000/50000 [==============================] - 682s - loss: 0.7704 - acc: 0.7394 - val_loss: 0.6468 - val_acc: 0.7891
Epoch 46/200
50000/50000 [==============================] - 681s - loss: 0.7698 - acc: 0.7391 - val_loss: 0.6166 - val_acc: 0.7910
Epoch 47/200
50000/50000 [==============================] - 672s - loss: 0.7672 - acc: 0.7399 - val_loss: 0.6231 - val_acc: 0.7876
Epoch 48/200
50000/50000 [==============================] - 674s - loss: 0.7517 - acc: 0.7440 - val_loss: 0.6103 - val_acc: 0.7957
Epoch 49/200
50000/50000 [==============================] - 677s - loss: 0.7681 - acc: 0.7384 - val_loss: 0.6192 - val_acc: 0.7906
Epoch 50/200
50000/50000 [==============================] - 675s - loss: 0.7691 - acc: 0.7413 - val_loss: 0.6612 - val_acc: 0.7835
Epoch 51/200
50000/50000 [==============================] - 669s - loss: 0.7586 - acc: 0.7429 - val_loss: 0.6632 - val_acc: 0.7771
Epoch 52/200
50000/50000 [==============================] - 672s - loss: 0.7548 - acc: 0.7449 - val_loss: 0.6844 - val_acc: 0.7724
Epoch 53/200
50000/50000 [==============================] - 671s - loss: 0.7605 - acc: 0.7431 - val_loss: 0.6271 - val_acc: 0.7911
Epoch 54/200
50000/50000 [==============================] - 670s - loss: 0.7607 - acc: 0.7427 - val_loss: 0.6207 - val_acc: 0.7934
Epoch 55/200
50000/50000 [==============================] - 667s - loss: 0.7570 - acc: 0.7445 - val_loss: 0.5973 - val_acc: 0.8035
Epoch 56/200
50000/50000 [==============================] - 666s - loss: 0.7657 - acc: 0.7407 - val_loss: 0.6121 - val_acc: 0.7956
Epoch 57/200
50000/50000 [==============================] - 665s - loss: 0.7652 - acc: 0.7424 - val_loss: 0.5917 - val_acc: 0.8051
Epoch 58/200
50000/50000 [==============================] - 666s - loss: 0.7587 - acc: 0.7435 - val_loss: 0.6507 - val_acc: 0.7855
Epoch 59/200
50000/50000 [==============================] - 662s - loss: 0.7495 - acc: 0.7471 - val_loss: 0.6322 - val_acc: 0.7917
Epoch 60/200
50000/50000 [==============================] - 662s - loss: 0.7574 - acc: 0.7463 - val_loss: 0.6830 - val_acc: 0.7764
Epoch 61/200
50000/50000 [==============================] - 663s - loss: 0.7506 - acc: 0.7442 - val_loss: 0.6091 - val_acc: 0.7978
Epoch 62/200
50000/50000 [==============================] - 658s - loss: 0.7609 - acc: 0.7454 - val_loss: 0.6281 - val_acc: 0.7934
Epoch 63/200
50000/50000 [==============================] - 657s - loss: 0.7482 - acc: 0.7492 - val_loss: 0.6026 - val_acc: 0.8025
Epoch 64/200
50000/50000 [==============================] - 653s - loss: 0.7471 - acc: 0.7489 - val_loss: 0.6273 - val_acc: 0.7913
Epoch 65/200
50000/50000 [==============================] - 660s - loss: 0.7507 - acc: 0.7452 - val_loss: 0.6090 - val_acc: 0.7979
Epoch 66/200
50000/50000 [==============================] - 656s - loss: 0.7471 - acc: 0.7474 - val_loss: 0.6177 - val_acc: 0.7970
Epoch 67/200
50000/50000 [==============================] - 655s - loss: 0.7515 - acc: 0.7453 - val_loss: 0.5881 - val_acc: 0.8052
Epoch 68/200
50000/50000 [==============================] - 661s - loss: 0.7476 - acc: 0.7479 - val_loss: 0.6323 - val_acc: 0.7899
Epoch 69/200
50000/50000 [==============================] - 667s - loss: 0.7466 - acc: 0.7486 - val_loss: 0.5878 - val_acc: 0.8070
Epoch 70/200
50000/50000 [==============================] - 708s - loss: 0.7435 - acc: 0.7515 - val_loss: 0.6329 - val_acc: 0.7951
Epoch 71/200
50000/50000 [==============================] - 731s - loss: 0.7546 - acc: 0.7462 - val_loss: 0.6165 - val_acc: 0.7941

**Comment by ChristianThomae** [comment]
Maybe related to changes to the configuration of the image augmentation generator here: https://github.com/fchollet/keras/commit/45714e343f36292cec938dc01ebb8a2dec10cea4
